packages/codegen-config/src/index.ts

```diff
@@ -10,6 +10,8 @@ const baseConfig: CodegenConfig = {
 				enumsAsTypes: true,
 				skipTypename: true,
 				useTypeImports: true,
+				useImplementingTypes: true,
```

Comment:
Is this supported with the `client` preset ? https://the-guild.dev/graphql/codegen/plugins/presets/preset-client

Reply:
Accidentally added when trying to fix fragment types. Removed